src/website/utils/DownloadData.tsx

```diff
@@ -0,0 +1,28 @@
+const downloadSensorData = async (sensorType) => {
+  try {
+    const response = await fetch(`http://localhost:3005/api/${sensorType}`);
```

Comment:
Please use the env vars when calling an API using our host url. Example can be found here: https://github.com/UBCSailbot/sailbot_workspace/blob/main/src/website/stores/GPS/GPSService.ts#L9